.github/workflows/test.yml

```diff
@@ -40,37 +40,43 @@ jobs:
     strategy:
       fail-fast: true
       matrix:
-        ghc: ["9.0.1", '8.10.7', '8.10.6', "8.8.4", "8.6.5"]
+        ghc: ["9.2.1", "9.0.1", '8.10.7', '8.10.6', "8.8.4", "8.6.5"]
         os: [ubuntu-latest, macOS-latest]
```

Comment:
I think now we can merge `windows-latest` into here, as we build HLS in Windows for all versions listed here

Reply:
PR is open, give it a go

Reply:
@pepeiborra Sorry, I don't understand what you mean. You mean there is a related PR, or you mean this PR should be independent from this minor issue?

Reply:
Ah you mean since this branch is open so I can fix that. Now I see it.

Reply:
yeah, good catch, i would do in a separate pr on master and then merge it